src/nanoemoji/nanoemoji.py

```diff
@@ -24,497 +24,135 @@
-            # scale x/y translation and flip y movement to match font space
-            transform = transform._replace(
-                e=transform.e * svg_units_to_font_units.a,
-                f=transform.f * svg_units_to_font_units.d,
-            )
-            glyph.components.append(
-                Component(baseGlyph=base_glyph.name, transformation=transform)
-            )
-    else:
-        # Not a composite, just draw directly on the glyph
-        draw_svg_path(painted_layer.path, glyph.getPen(), svg_units_to_font_units)
-
-    ufo.glyphOrder += glyph_names
-
-    return glyph
-
-
-def _draw_glyph_extents(ufo: ufoLib2.Font, glyph: Glyph):
-    # apparently on Mac (but not Linux) Chrome and Firefox end up relying on the
-    # extents of the base layer to determine where the glyph might paint. If you
-    # leave the base blank the COLR glyph never renders.
-
-    # TODO we could narrow this to bbox to cover all layers
-
-    pen = glyph.getPen()
-    pen.moveTo((0, 0))
-    pen.lineTo((ufo.info.unitsPerEm, ufo.info.unitsPerEm))
-    pen.endPath()
-
-    return glyph
-
-
-def _glyf_ufo(ufo, color_glyphs):
-    # glyphs by reuse_key
-    glyphs = {}
-    for color_glyph in color_glyphs:
-        logging.debug(
-            "%s %s %s",
-            ufo.info.familyName,
-            color_glyph.glyph_name,
-            color_glyph.transform_for_font_space(),
-        )
-        parent_glyph = ufo.get(color_glyph.glyph_name)
-        for painted_layer in color_glyph.as_painted_layers():
-            # if we've seen this shape before reuse it
-            reuse_key = _inter_glyph_reuse_key(painted_layer)
-            if reuse_key not in glyphs:
-                glyph = _create_glyph(color_glyph, painted_layer)
-                glyphs[reuse_key] = glyph
-            else:
-                glyph = glyphs[reuse_key]
-            parent_glyph.components.append(Component(baseGlyph=glyph.name))
-
-        # No great reason to keep single-component glyphs around
-        if len(parent_glyph.components) == 1:
-            component = ufo[parent_glyph.components[0].baseGlyph]
-            del ufo[component.name]
-            ufo[color_glyph.glyph_name] = component
-            assert component.name == color_glyph.glyph_name
-
-
-def _colr_paint(colr_version: int, paint: Paint, palette: Sequence[Color]):
-    # For COLRv0, paint is just the palette index
-    # For COLRv1, it's a data structure describing paint
-    if colr_version == 0:
-        # COLRv0: draw using the first available color on the glyph_layer
-        # Results for gradients will be suboptimal :)
-        color = next(paint.colors())
-        return palette.index(color)
-
-    elif colr_version == 1:
-        # COLRv1: solid or gradient paint
-        return paint.to_ufo_paint(palette)
 
-    else:
-        raise ValueError(f"Unsupported COLR version: {colr_version}")
+def picosvg(input_svg: str) -> str:
```

Comment:
I find the name of this function not very clear, also I fear it may shadow the `picosvg` module name.

Reply:
Updated to picosvg_dest